app/experimenter/static/rapid/components/experiments/ExperimentDetails.tsx

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import { useHistory } from "react-router";
+
+import { useExperimentState } from "experimenter-rapid/contexts/experiment/hooks";
+
+const LabelledRow: React.FC<{ label: string; value?: string | undefined }> = ({
+  children,
+  label,
+  value,
+}) => {
+  return (
+    <div className="row my-3">
+      <strong className="col-2 d-inline-block pt-2">{label}</strong>
```

Comment:
Should these be `<label>` elements, or i guess a11y nit-picking might not matter here since these are just read-only confirmation fields.